src/main.ts

```diff
@@ -82,13 +92,14 @@ export async function run() {
     const matchersPath = path.join(__dirname, '../..', 'matchers.json');
     core.info(`##[add-matcher]${matchersPath}`);
 
-    // output the version actually being used
-    core.info(goVersion);
-
-    core.setOutput('go-version', parseGoVersion(goVersion));
+    core.setOutput('go-version', parsedGoVersion);
+    core.setOutput('go-path', goEnvJson['GOPATH']);
+    core.setOutput('go-root', goEnvJson['GOROOT']);
+    core.setOutput('go-cache', goEnvJson['GOCACHE']);
+    core.setOutput('go-mod-cache', goEnvJson['GOMODCACHE']);
+    core.setOutput('go-env', goEnvJson);
```

Comment:
Hello @lucacome 

Can you please remove the most outputs except these 3 ones

```
    core.setOutput('go-version', parsedGoVersion);
    core.setOutput('go-cache', goEnvJson['GOCACHE']);
    core.setOutput('go-mod-cache', goEnvJson['GOMODCACHE']);
```

The others does not seem relate to any real-world use cases.

Reply:
I'm still of the opinion that the entire `go env -json` should be exposed in its entirety - I find most of the vars useful.